docs/Troubleshooting.md

```diff
@@ -0,0 +1,159 @@
+# Troubleshooting
+
+## Not large enough perf_event_mlock_kb
```

Comment:
It's not great that title repeats or rephrases the actual message.
I expects users to look for the original message. Maybe titles are not needed?

Reply:
Sure Andrei I will reformat into a list list removing headers

Reply:
Addressed in [e6282c8](https://github.com/async-profiler/async-profiler/pull/1029/commits/e6282c83bc0aa6007df310fd5e638e9d4d5ed6f1)